src/FileExplorer/index.jsx

```diff
@@ -1,3 +1,5 @@
+/* global gtag */
```

Comment:
What is the purpose of this?

Reply:
https://eslint.org/docs/user-guide/configuring#specifying-globals